city_scrapers/spiders/il_port_district.py

```diff
@@ -79,7 +79,7 @@ def parse_schedules(self, response):
 
                 agendas_links = (
                     self.agendas_dict.get((classification, start.strftime('%B %Y')), []) +
-                    self.agendas_dict.get((classification, start.strftime('%-m-%y')), [])
+                    self.agendas_dict.get((classification, start.strftime('%m-%y')), [])
```

Comment:
This should be kept out of this PR, it may be causing issues due to platform compatibility but it's easier to review if we keep the changes limited

Reply:
Yes, I'll remove it